server/server.ts

```diff
@@ -179,6 +184,11 @@ export const AppRoutes = app
   .get("/chat/trace", zValidator("query", chatTraceSchema), GetChatTraceApi)
   // this is event streaming end point
   .get("/message/create", zValidator("query", messageSchema), MessageApi)
+  .get(
```

Comment:
instead of separate route, put an if condition in the `MessageApi` itself. refer to #106